Cargo.toml

```diff
@@ -91,10 +91,21 @@ lmdb-rkv = { git = "https://github.com/mozilla/lmdb-rs", rev = "df1c2f5" }
 serde_cbor = { git = "https://github.com/mobilecoinofficial/cbor", rev = "4c886a7c1d523aae1ec4aa7386f402cb2f4341b5" }
 
 # Not-yet-released version that depends on newer crates.
-bulletproofs = { git = "https://github.com/dalek-cryptography/bulletproofs", rev = "464acb72432ccc6c5131397fd811a5f01ec2454b" }
+bulletproofs = { git = "https://github.com/eranrund/bulletproofs", rev = "8a7c9cdd1efafa3ad68cd65676302f925de68373" }
```

Comment:
We should host these on `mobilecoinoficial`

Reply:
I asked about this at some point and the conclusion was that it is not worth it since @isis-mc is working on official uprevs that should be available sometime soon.
@isis-mc do you have a timeline for that? Thanks!